### ISSUE TITLE
Adding push notifications to web app to send to Android app

### ISSUE THREAD

**Issue by MichaelBujard** [MichaelBujard/DineRush#4]
Hi,

I am learning how to add push notifications to our app.

One way to do this is through Cloud Messaging.

This would mean that I add Firebase to your Android app.

To add Firebase to the Android app, I would first register the app, then download the config file, then add the Firebase SDK, and finally run the app to verify installation.

Is this the right way to add push notifications to the DineRush/Final Project web app as mentioned in the last Issue?

Is there another way to do this, or is this the correct way to do this?

**Comment by rayanazzam** [comment]
When using firebase for the moblie app, I already had to go through all the setups you mentioned. So to be able to send notifications via the web app to the user, we would need a message instance for each particular user that we can both update from each side( i.e the web app, and the mobile app)

**Comment by MichaelBujard** [comment]
OK, so does that mean we store the message instance as a variable belonging to the user, so each user has a message, as in I create a User js object with a messaage field?

**Comment by rayanazzam** [comment]
Yes, either that if we want to allow communication at all times, or add a message field to an order object, so users communicate during an order. Both could work. 

Best,
Rayan

**Comment by MichaelBujard** [comment]
I think adding a message field to an order object is better. Thank you for the feedback!

Mike

**Comment by rayanazzam** [comment]
I think we should set up a skype meeting sometime this weekend to discuss
the overall architecture and also break up the task efficiently. Let me
know what time works for you. I am essentially free all day tomorrow and
later tonight.

thanks,
Rayan

**Comment by MichaelBujard** [comment]
How about tomorrow at 4PM?

Mike

**Comment by rayanazzam** [comment]
Yeah, that works for me as well.
